web/src/lib/components/faces-page/unmerge-face-selector.svelte

```diff
@@ -126,6 +146,19 @@
     </svelte:fragment>
     <svelte:fragment slot="trailing">
       <div class="flex gap-4">
+        <Button
+          title={'Unassign selected assets to a new person'}
+          size={'sm'}
+          disabled={disableButtons || hasSelection}
+          on:click={handleUnassign}
+        >
+          {#if !showLoadingSpinnerUnassign}
+            <Icon path={mdiTagRemove} size={18} />
+          {:else}
+            <LoadingSpinner />
+          {/if}
```

Comment:
Nit: If you already call the variable `showLoadingSpinnerUnassign` I'd just have it non-negated and show the loading spinner if`showLoadingSpinnerUnassign`